service/initializer/protocol_state_test.go

```diff
@@ -94,6 +94,7 @@ func TestProtocolState(t *testing.T) {
 
 		reader := bytes.NewBuffer(data)
 
-		err = initializer.ProtocolState(reader, db)
+		_ = initializer.ProtocolState(reader, db)
```

Comment:
you may consider `require.NoError` instead.

Reply:
it panics on this line, so I saved a heap alloc